packages/subgraph/schema.graphql

```diff
@@ -1757,6 +1815,11 @@ type Pool @entity {
     totalAmountInstantlyDistributedUntilUpdatedAt: BigInt!
     totalAmountFlowedDistributedUntilUpdatedAt: BigInt!
     totalAmountDistributedUntilUpdatedAt: BigInt!
```

Comment:
it's preferrable to get this value from the RPC now for a pool member, but it's beneficial to write out how the new fields change the formula on the client-side though